apps/anoma_lib/lib/anoma_glossary.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Glossary do
+  use GlossaryBuilder
+
+  define transaction(candidate) do
+    """
```

Comment:
let's keep this as `define transaction_candidate` to avoid the ()'s and make it easier to link